packages/core/integration-tests/test/fs.js

```diff
@@ -52,7 +52,7 @@ describe('fs', function() {
       assert.equal(output, 'hello');
     });
 
-    it('should inline a file with fs require destructure', async function() {
+    it.skip('should inline a file with fs require destructure', async function() {
```

Comment:
This is related to babel issues afaik, skipped it for now as babel was locked to <7.4 due to bugs.

Reply:
> locked to <7.4 due to bugs.

How long will it stay it that way?

Reply:
Not sure Devon locked it, perhaps the bugs have already been fixed

cc @devongovett

Reply:
One issue atm with the old babel: `This experimental syntax requires enabling the parser plugin: 'objectRestSpread'` if the deps contain rest spread (which isn't a proposal anymore)

Reply:
I believe babel is fixed now, so we can upgrade it again.